test/unit/firebase-namespace.spec.ts

```diff
@@ -48,7 +48,7 @@ import {
   v1beta1,
   setLogFunction,
 } from '@google-cloud/firestore';
-import { InstanceId } from '../../src/instance-id/instance-id';
+import { InstanceId } from '../../src/instance-id/instance-id-internal';
```

Comment:
Here too.

Reply:
Fixed, though I ended up using both the interface and implementation for this:

https://github.com/firebase/firebase-admin-node/blob/560f1061669b6172f5f07c656d5934cae4f65c9a/test/unit/firebase-namespace.spec.ts#L626-L628